src/View/Components/Badge.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace WireUi\View\Components;
+
+class Badge extends BaseBadge
+{
+    public function getPulse(): void
+    {
+        if ($this->info) {
+            $this->setPulseAttributes('bg-info-500', 'bg-info-400');
+        }
+        if ($this->warning) {
+            $this->setPulseAttributes('bg-warning-500', 'bg-warning-400');
+        }
+        if ($this->positive) {
+            $this->setPulseAttributes('bg-positive-500', 'bg-positive-400');
+        }
+        if ($this->negative) {
+            $this->setPulseAttributes('bg-negative-500', 'bg-negative-400');
+        }
+    }
+
+    private function setPulseAttributes($pulseColor, $pingColor)
+    {
+        $this->pulseColor = $pulseColor;
+        $this->pulsePingColor = $pingColor;
+    }
+
+    public function getSize(bool $lg): void
+    {
+        if ($lg) {
```

Comment:
And the configuration of the other sizes?

Reply:
done added sm, md,  lg